File: src/test/java/io/github/stscoundrel/javatemplate/AppTest.java
package io.github.stscoundrel.javatemplate;

import junit.framework.Test;
import junit.framework.TestCase;
import junit.framework.TestSuite;

/**
 * Unit test for simple App.
 */
public class AppTest 
    extends TestCase
{
    /**
     * Smoke tests for test suite.
     */
    public void testCasesRun()
    {
        assertTrue( true );
        assertFalse( false );

        assertEquals(App.dummyMethod(2, 2), 4);
    }
}

File: src/test/scala/io/github/stscoundrel/scalatemplate/AppTest.scala
package io.github.stscoundrel.scalatemplate

import org.scalatest.funsuite.AnyFunSuite

class AppTest extends AnyFunSuite {
  test("test cases run") {
    assert(true)
    assert(!false)
    assert(App.dummyMethod(2, 2) == 4)
  }
}